composer.json

```
@@ -44,12 +44,13 @@
     "phpmd/phpmd": "^2.8",
     "phpunit/phpunit": "^7.5|^8.4|^9.5",
     "psalm/plugin-phpunit": "^0.15",
-    "symfony/browser-kit": "^5.4|^6.0",
-    "symfony/framework-bundle": "^5.4|^6.0",
-    "symfony/phpunit-bridge": "^5.4|^6.0",
-    "symfony/yaml": "^5.4|^6.0",
+    "symfony/browser-kit": "^5.4|~6.0",
```

You can drop 5.4 there

---

```suggestion
    "symfony/browser-kit": "^6.0",
```

(With conflicts as well)